### ISSUE TITLE
[Feature] Can the OHLC data of candlestick chart like tradigview.com be added?

### ISSUE THREAD

**Issue by suncg-sgla** [apache/echarts#18519]
### What problem does this feature solve?

Pic below is a screenshot from tradingview, as we can see, the OHLC and volume is shown at a fixed position, what we can do now in Echarts is a tooltip box floating on the chart, so can fixed positon OHLC and volume data feature be add in Echarts?

![2023-04-17_11-06](https://user-images.githubusercontent.com/105915514/232368887-3d66cbc9-e99e-46e8-95b1-0fa2bfc335e3.png)

### What does the proposed API look like?

A fixed banner contains OHLCV data!

**Comment by helgasoft** [comment]
yes, possible with [graphic.text](https://echarts.apache.org/en/option.html#graphic.elements-text.type) and [mouseover](https://echarts.apache.org/en/api.html#events.Mouse%20events.mouseover)
[Demo Code](https://echarts.apache.org/examples/en/editor.html?c=candlestick-simple&code=PYBwLglsB2AEC8sDeAoWsAeBBDEDOAXMmurACYCGYFRA2gOQBMADAIwDsAtK854wCz0ANLCZsuPPgFZholh269GANlliFkxu3oBdEgF8hJAJ458RJIZJ4ApgCcINwrFol0qUqTDGQNovQBjCmgyABsnSACAa2E3UkpqOjjPWhYRAGZ-ER4MgA4dI08U_mYMqQzS2ClmAuTSWnTWPIz0kX5-WqL69Nzs8qq2xj0u4dJ9Ek7YAHM7ChAACwgApM8PIu9ffzAbDDBYrocp-bB_AFsIMjCbfaK8b3CLWG3drZ292HHPT9g9fQBuFCnYwAYXmFDsYAAdDAABT0U7AACutmAADd7GpbA4nLIYSAAJQIAB8xHQoDAiDW6Bmc0WyxcVK8Pj8omee0Kt3uLKQTzeRBAkNRFFCiJsHzi-lG3yBoPBUNsYAA8uAoNAYeT8QD9JqgA)
